gensim/models/translation_matrix.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import warnings
+import numpy as np
+
+from collections import OrderedDict
+from gensim import utils
+from six import string_types
+
+
+"""
+Produce translation matrix to translate the word from one language to another language, using either
+standard nearest neighbour method or globally corrected neighbour retrieval method [1].
```

Comment:
Could note that this is potentially more useful than just word-to-word translation; it'll work for any two sets of named-vectors where there are some paired-guideposts to learn the transformation.